site/docs/guidelines/intercom.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Intercom (Live chat)
+navTitle: Intercom (Live chat)
+summaryParagraph: Intercom (Live chat) is used to allow users to quickly access help or to send targeted messages to users.
+tags: ["Intercom", "Live chat"]
+needToKnow:
+  - It is a fixed element, and it it should appear in the same position across the product 
+  - It should not block anything on the page behind it.
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+<iframe style="border: none;" width="800" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D53304%253A47849&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+### When to use
```

Comment:
No need for this heading here, we can cut straight to the intro description (and reserve the "when to use" heading for the section below, `## When to use and when not to use`)